console/src/components/Error/ErrorBoundary.tsx

```diff
@@ -60,9 +69,20 @@ class ErrorBoundary extends React.Component {
     );
   }
 
+  initialState = {
```

Comment:
Can you move it outside of the component and use `ErrorBoundaryState` to add a type to it, as it's not dependent on component props or state?

Reply:
No, there are no dependencies. I can move it outside of the component.